Day1/puzzle_one.py

```diff
@@ -1,8 +1,6 @@
 
-expense_list = []
 with open("day_one.txt", 'r') as puzzle_input:
-    for line in puzzle_input:
-        expense_list.append(int(line))
```

Comment:
Where you can, it's good to avoid exposing mutable state to the outside world. There's just more scope for errors if you allow the outside world to see `expense_list` when it's in a half-built state.